src/input/keyboard.c

```diff
@@ -38,6 +38,8 @@ struct keyinfo {
 
 static bool should_cancel_cycling_on_next_key_release;
 
+struct keybind *cur_keybind;
```

Comment:
This should be static.

I am also not sure about `Reconfigure` happening between press and release, this seems to miss some reconfigure hook to reset `cur_keybind` in that case.

Reply:
Done